.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "3.6"
+
+install:
+  - pip install -r requirements.txt
+  - pip install -r optional-requirements.txt
+
+before_script:
+  - export PYTHONPATH=$PYTHONPATH:$(pwd)
+script:
+  - cd tests
+  - pytest
```

Comment:
does OpenML have a Travis account that allows us to use it for this repo? nice!

Reply:
Yes, I enabled it through the openml github organisation 👍